src/main/java/com/ericsson/ei/mongo/MongoConstants.java

```diff
@@ -5,4 +5,10 @@
     public static final String EVENT = "Event";
     public static final String TIME = "Time";
     public static final String NOT_LOCKED = "0";
+
+    public static final String DESTINATION = "destination";
```

Comment:
Now that I think about, these keys are rabbitmq specific right, so they should probably not be in this package but perhaps placed in a rabbitmq package instead.

Since no rabbitmq package exists today I think it's a bit outside of the scope of this PR so keep the constans as they are for now. We can write an issue to refactor the rabbitmq related files into a separate package, and move these constants there also, at a later time.

Reply:
You could add "MB_" in front of each constants anyway, so it becomes clear that its relates to MessageBus, if you don't want to move it now to MessageBusContants/RabbtiMqConstants class.